packages/entities/entities-routes/src/components/RouteForm.cy.ts

```diff
@@ -253,6 +253,7 @@ describe('<RouteForm />', { viewportHeight: 700, viewportWidth: 700 }, () => {
           cy.get('.expression-editor .monaco-editor').should('be.visible')
 
           // base advanced fields
+          cy.getTestId('collapse-trigger-content').click()
```

Comment:
Question: why do we need to click this element?

Reply:
This is because previously, the "Traditional" and "Expressions" tabs shared the same container for "Advanced fields", but this was changed in this PR, where each tab will have its own "Advanced fields" container and the collapsed state. Therefore, we will have to click the "Advanced fields" again to show these fields after switching to another tab.

Reply:
I see. Thanks for the explanation!